frontend/lib/src/components/elements/ArrowVegaLiteChart/arrowUtils.ts

```diff
@@ -86,7 +79,7 @@ export interface WrappedNamedDataset {
 export function getInlineData(
   quiverData: Quiver | null
 ): { [field: string]: any }[] | null {
-  if (!quiverData || quiverData.data.numRows === 0) {
+  if (!quiverData || quiverData.dimensions.dataRows === 0) {
```

Comment:
is `quiverData.dimensions.dataRows` a number / count? If yes, it might make sense to rename it to `quiverData.dimensions.dataRowCount` or similar

Reply:
Yeah, I wanted to rename it a couple of times... e.g. prefix all variables with `num` ar suffix with `Count`. But haven't done so yet since its already under the `dimensions` object. But it probably would make code more readable if it has an additional indication that its just a number.

Reply:
I updated the namings